api/src/test/java/io/opentelemetry/trace/DefaultSpanTest.java

```diff
@@ -45,6 +45,7 @@ void doNotCrash() {
     span.addEvent("event", 0);
     span.addEvent("event", Attributes.of(booleanKey("MyBooleanAttributeKey"), true));
     span.addEvent("event", Attributes.of(booleanKey("MyBooleanAttributeKey"), true), 0);
+    span.setStatus(StatusCanonicalCode.OK);
```

Comment:
should we also have one with a description?

Reply:
Yeah probably - many cases of null, e.g null code, seem to be missing from this test too, so not sure what it's meant to test TBH. I'm relatively happy with the javadoc changes here but am fine with merging either this or #1754 to get a fix for the signature in!